tensornetwork/backends/tensorflow/tensorflow_backend_test.py

```diff
@@ -515,6 +515,26 @@ def test_matmul():
   np.testing.assert_allclose(expected, actual)
 
 
+@pytest.mark.parametrize("dtype", tf_dtypes)
+def test_abs(dtype):
+  shape = (4, 3, 2)
```

Comment:
seeds, can you check every test and add seeds to all of them? thanks

Reply:
Ok, I think (hope?) I got them all